front-end/src/components/banner/player-champ-banner.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Box, Typography } from '@mui/material';
+import ImgBox from '../risen-box/img-box';
+import AggregatedPlayerStatModel from '../../../../Common/models/aggregatedplayerstat.model';
+import { roundTo } from '../../../../Common/utils';
+
+export interface PlayerChampBannerItem {
+  stats: AggregatedPlayerStatModel;
+  championPortraitId: number;
+}
+
+export interface PlayerChampBanners {
+  top: PlayerChampBannerItem;
+  jungle: PlayerChampBannerItem;
+  mid: PlayerChampBannerItem;
+  bot: PlayerChampBannerItem;
+  supp: PlayerChampBannerItem;
+}
+
+interface IndividualBoxProps {
+  roleInfo: PlayerChampBannerItem;
+  position: string;
+}
+
+export default function PlayerChampBanner(props: PlayerChampBanners) {
+  const IndividualBox = ({ roleInfo, position }: IndividualBoxProps) => {
+    const {
+      championPortraitId,
+      stats,
+    } = roleInfo;
+    return (
+      <Box sx={{ height: 500 }}>
+        <Box>
+          <Typography align='left' fontFamily="Montserrat">{position}</Typography>
+        </Box>
+        <Box sx={{ height: 300, overflow: 'hidden', width: 190, clipPath: 'polygon(0 0, 75% 0, 100% 100%, 25% 100%)', backgroundImage: `url(/images/champions/profile/${championPortraitId}_0.jpg)`, backgroundSize: 'cover', backgroundPosition: 'center' }}>
+          {/* <ImgBox
+            src={'/images/champions/splash/1_0.jpg'}
+            height={'400px'}
+          /> */}
```

Comment:
Commented out code, should remove if we dont need it

Reply:
Fixed